src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyExecutionManager.cs

```diff
@@ -37,6 +37,12 @@ internal class ProxyExecutionManager : ProxyOperationManager, IProxyExecutionMan
         private bool skipDefaultAdapters;
         private readonly IFileHelper fileHelper;
 
+        // Only send coverlet inproc datacollector dll to be initialized via testhost, 
+        // ideally this should get initialized via InProcessDC Node in runsettings, but 
+        // somehow it is failing, hence putting this ugly HACK, to fix issues like
+        // https://developercommunity.visualstudio.com/content/problem/738856/could-not-load-file-or-assembly-microsoftintellitr.html
```

Comment:
In the runsettings, we do not send the complete codebase path for coverlet. We send just the dll name (coverlet.collector.dll). Later, in test host while initializing the inproc data collector, we look for all directories from where (datacollector)extensions were loaded and try to find the coverlet.collector.dll in those paths. (https://github.com/microsoft/vstest/blob/master/src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/InProcDataCollectionExtensionManager.cs#L69)

If there's a way we can add the full codebase in runsettings, then this can be avoided.